doc/document_format/document_format_v1.md

```diff
@@ -0,0 +1,95 @@
+- `start`: The start of the `text` in the associated audio in seconds. It MUST be `null` or a floating point number. It MUST be `null` if and only if the `end` property is also `null`. If not `null`, it MUST be greater to or equal 0. If not `null`, it MUST be smaller to or equal to the `end` property.
+- `end`: The end of the `text` in the associated audio in seconds. It MUST be `null` or a floating point number. It MUST be `null` if and only if the `start` property is `null`. If not `null`, it MUST be greater to or equal to the `start` property. If not `null`, it MUST be smaller than or equal to the total length of the associated audio.
+
+## Speaker Names
+
+A transcribee v1 document MAY contain a mapping of speaker ids to strings in the `speaker_names` property.
+If this property exists, it MUST be an object.
+
+Each key of this object MUST be an integer representing a speaker id.
+The speaker id SHOULD be used as a `speaker` in the `alternative_speakers` array of a paragraph in the transcribee v1 document.
+
+Each value of this objcet MUST be a string.
+It MAY NOT be the empty string.
+It MAY NOT consist of only white space (as defined in [ECMAScript Section `TrimString`](https://tc39.es/ecma262/#sec-trimstring)).
+It MAY otherwise contain the entire range of unicode characters.
+
+## Diarization Data
+
+A transcribee v1 document MAY information on which speakers speaks where in the associated audio in the `diarization` property.
+If this property exists, it MUST be an array.
+The array MAY be empty.
+
+Each element of the array MUST be an object with the following properties:
+
+- `start`: The start of this segment in the associated audio in seconds. It MUST be a floating point number
+- `end`: The end of this segment in the associated audio in seconds. It MUST be a floating point number
+- `speakers`: The detected speakers in this segment. It MUST be an array of integers representing the speaker ids of the speakers in this segment.
+
+The segments MAY overlap.
+These MAY be times in the associated audio which overlap with no segment.
+
+## Invariants
+
+There are a number of restrictions / invariants that a transcribee v1 document MUST conform to.
+
+Note: Some of them follow directly from the specification above but were deemed important enough to list them again here.
+
+- The `start` property of a _text atom_ is strictly increasing
+- $\forall atom \in atoms: atom.start \neq null \iff atom.end \neq null$
+- $\forall atom \in atoms: atom.start\neq null \implies atom.start \leq atom.end$
+
+### Invalid Assumptions
+
+There are a few assumptions that _seem_ valid at first glance, but are not always the case.
+The followling list contains some of them.
+The list is not normative and may be extended at any time.
+
+- For every point in time there is at least one atom: $\forall t \in [0, audio.length]: \exists atom \in atoms: (atom.start >= t) \wedge (atom.end <= t)$
```

Comment:
can I have this in text form?

Reply:
The text form is: `For every point in time there is at least one atom`

Reply:
I think this is much more readable than the math notation